tinnsleep/scoring.py

```diff
@@ -111,6 +109,25 @@ def burst_to_episode(burst_list, delim=3):
 
     return ep_list
 
+def generate_annotations(li_ep):
+    """ Transforms a chronological list of episodes into a
+        chronological list of annotations
+
+        Parameters
+        ----------
+        li_ep : list of episode instances
+
+        Returns
+        -------
+        Annotations : Chronological list of annotations
+        """
+
+    annotations = []
+    li_ep = rearrange_chronological(li_ep)
```

Comment:
not really important to rearrange (Annotations can be in fact non-chronological without any problem, it may be the case when you stack many different type of annotations build by different algorithms)

Please remove (use can use by himself rearrange if needed before)

Reply:
Remember that any type of sorting can be in fact quite computationally heavy when there are several thunsands event (which is in fact the case)